### PR TITLE
Add new containsAtLeast / Most(iterable,count,value) methods

### PR DESCRIPTION
**Pull request by Skezzowski**

### CODE REVIEW

File: src/main/java/com/github/underscore/U.java [hunk 1]
```diff
@@ -621,6 +621,34 @@ public boolean contains(final T elem) {
...
+
+    public boolean containsAtMost(final Integer count, final T value) {
+        return U.containsAtMost(this.iterable, count, value);
+    }
+
+    public static <E> boolean containsAtLeast(final Iterable<E> values, final Integer count, final E value) {
+        List<E> filtered = filter(values, new Predicate<E>() {
+            @Override
+            public boolean test(E input) {
+                return input == value;
```
Comment (javadev) [inline src/main/java/com/github/underscore/U.java]: It should be `equals` check. `return elem == null ? e == null : elem.equals(e);`

File: src/main/java/com/github/underscore/U.java [hunk 2]
```diff
@@ -621,6 +621,34 @@ public boolean contains(final T elem) {
         return contains(list.subList(fromIndex, list.size()), elem);
     }
 
+    public boolean containsAtLeast(final Integer count, final T value) {
```
Comment (javadev) [inline src/main/java/com/github/underscore/U.java]: It may be `iterable, value, count` parameters order and `count` type is `int` not `Integer`.

File: src/test/java/com/github/underscore/CollectionsTest.java [hunk 3]
```diff
@@ -1031,6 +1030,22 @@ public void contains() {
         assertFalse(result3);
         final boolean result4 = U.contains(asList(1, 2, null), null);
         assertTrue(result4);
+        final boolean result5 = U.containsAtLeast(asList(1, 2, 2), 2, 2);
```
Comment (javadev) [inline src/test/java/com/github/underscore/CollectionsTest.java]: It may be unit test with `null` inside array.

File: src/test/java/com/github/underscore/CollectionsTest.java [hunk 4]
```diff
@@ -1031,6 +1030,26 @@ public void contains() {
...
+        assertTrue(result9);
+        final boolean result10 =  new U<Integer>(asList(1, 2, 2)).containsAtLeast(2, 3);
+        assertFalse(result10);
+        final boolean result11 = new U<Integer>(asList(1, 2, 2)).containsAtMost(3, 2);
+        assertTrue(result11);
+        final boolean result12 =  new U<Integer>(asList(1, 2, 2)).containsAtMost(2, 1);
+        assertFalse(result12);
+        final boolean result13 =  U.containsAtMost(asList(null, null, 2), null, 2);
+        assertTrue(result13);
+        final boolean result14 =  U.containsAtLeast(asList(null, null, 2), null, 2);
```
Comment (javadev) [inline src/test/java/com/github/underscore/CollectionsTest.java]: Good. We have a test case `null` in list and `null` value. It may be one more test case `null` in list and not null value.

File: src/test/java/com/github/underscore/CollectionsTest.java [hunk 5]
```diff
@@ -1031,6 +1030,30 @@ public void contains() {
         assertFalse(result3);
         final boolean result4 = U.contains(asList(1, 2, null), null);
         assertTrue(result4);
+        final boolean result5 = U.containsAtLeast(asList(1, 2, 2), 2, 2);
+        assertTrue(result5);
+        final boolean result6 = U.containsAtLeast(asList(1, 2, 2), 2, 3);
```
Comment (javadev) [inline src/test/java/com/github/underscore/CollectionsTest.java]: It will be good to have a separate unit test for each method.

**Comment by javadev** [comment]
LGTM
